docs/release_guide.md

```diff
@@ -1,43 +1,45 @@
-
-**Note:** Steps 6, 7, 9, 11 can be performed at the same time (no need to wait for one to finish before starting the other).
+Throughout the guide, `X.X.X` will mark the upcoming version.
+
+## Pre-release validations
+
+1) Branch out from the latest SDK PR used for a nightly test, name the branch `X.X.X`.
+2) Make sure the **CHANGELOG.md** file is in order. Add a subtitle `## X.X.X` under the `## Unreleased`, leaving the unreleased list empty.
+3) Run `poetry update` (in the SDK folder).
+5) Run `poetry version X.X.X`.
+6) Make sure that all the following passed:
+  - [content nightly](https://code.pan.run/xsoar/content/-/pipeline_schedules)
+  - [SDK nightly](https://code.pan.run/xsoar/content/-/pipeline_schedules)
+  - [SDK master](https://github.com/demisto/demisto-sdk) (last post-merge build passed)
+  - [content-gold](https://code.pan.run/xsoar/content-internal-dist/-/pipeline_schedules)
+  - [content-private](https://github.com/demisto/content-private/actions).
+  
+  If any failed, consult with the SDK owner, and see [Triggering nightlies manually](#triggering-nightlies-manually)
 
 ### Release process
 
-1) Click [Here](https://github.com/demisto/demisto-sdk/releases/new) (alternatively: visit the [SDK github page](https://github.com/demisto/demisto-sdk), click on **releases**, and then **Draft a new release**)
+1) Click [Here](https://github.com/demisto/demisto-sdk/releases/new)
 2) Set **Tag** and **Release title** to be `vX.X.X`.
 3) Select the SDK release branch as the **Target**.
 4) In the **Describe the release** text box, paste the `CHANGELOG` contents for this release.
-5) Click **Publish release**. Your release will go through a deploy build (follow it on the [CI website](https://app.circleci.com/pipelines/github/demisto/demisto-sdk). If the build is successful, and `vX.X.X` shows in [PyPi](https://pypi.org/project/demisto-sdk/), your release is public! 🎉
-6) Update [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) in a new branch, using the newly-released version, e.g `demisto-sdk==x.x.x`. **NOTE**: it may take up to an hour for our internal PyPi to sync with the real PyPi.
-7) Wait for the build to finish, or force merge your PR to the Content repository.
-8) Update **CHANGELOG.md** file - change the `# Changelog` header to the release version in the format `# X.X.X` e.g. `# 1.0.0`, and create a new `# Changelog` header at the top of the file.
+5) Click **Publish release**. Your release will go through a deploy build (follow it on the [CI website](https://app.circleci.com/pipelines/github/demisto/demisto-sdk).
+6) If the build is successful, and `vX.X.X` shows in [PyPi](https://pypi.org/project/demisto-sdk/), your release is public! 🎉
+7) Under the **Content** repo, run `poetry update`, make sure the SDK version is updated.  **NOTE**: it may take up to an hour for our PyPi mirror to sync with the real PyPi version, so the build may fail.
```

Comment:
I think it's better to run `poetry add demisto-sdk==<version> && poetry update`.
This will change the `demisto-sdk` requirement in `pyproject.toml` to the latest version.

If we don't do that, we can't be sure that the sdk version was really updated in lock because of conflicts.
In addition, I think we should add a step about resolving conflicts when there are conflicts when running this command.